foreman/foreman.spec

```diff
@@ -758,6 +766,11 @@ install -d -m0755 %{buildroot}%{_localstatedir}/lib/%{name}/tmp/pids
 install -d -m0755 %{buildroot}%{_localstatedir}/run/%{name}
 install -d -m0750 %{buildroot}%{_localstatedir}/log/%{name}
 install -d -m0750 %{buildroot}%{_localstatedir}/log/%{name}/plugins
+#Copy init scripts and sysconfigs
+install -Dp -m0644 %{SOURCE10} %{buildroot}%{_sysconfdir}/%{executor_service_name}
+install -Dp -m0644 %{SOURCE11} %{buildroot}%{_unitdir}/%{executor_service_name}.service
+mkdir -p %{buildroot}%{_bindir}
+ln -sv %{gem_instdir}/bin/dynflow-executor %{buildroot}%{_bindir}/%{executor_service_name}
```

Comment:
I wonder why this symlink needed because the unit file uses ```/usr/bin/dynflow-executor```. Could that be the issue?

Reply:
The gem_instdir doesn't expand so the symlink is broken:

```
[root@pipeline-katello-nightly-centos7 sysconfig]# ls /usr/bin/dynflowd -l
lrwxrwxrwx. 1 root root 35 Nov 13 20:58 /usr/bin/dynflowd -> %{gem_instdir}/bin/dynflow-executor
```